google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -275,7 +275,9 @@ public void sendAckOperations(
         requestObserver.onNext(request);
       }
     } catch (Exception e) {
-      logger.log(Level.WARNING, "failed to send acks", e);
+      if (isAlive()) {
+        logger.log(Level.WARNING, "failed to send ack operations", e);
+      }
```

Comment:
Maybe log at a verbose level if it's not alive?

Reply:
Done